### PR TITLE
MWPW-136802: CaaS Configurator UI controls update (Part II)

### PR DESCRIPTION
**Pull request by cmiqueo**
This PR adds the ability to collapse config panel to the CaaS configurator tool to allow for a larger view of the content.

image
 

Resolves: [MWPW-136802](https://jira.corp.adobe.com/browse/MWPW-136802)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/tools/caas?martech=off
- After: https://mwpw-136802--milo--adobecom.hlx.page/tools/caas?martech=off

### CODE REVIEW

**Review by sanrai — APPROVED** [review]

File: libs/blocks/caas/caas.css [hunk 1]
```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
   align-self: baseline;
   margin-top: 6px;
 }
+
+
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:532]: Extra new line can be removed

File: libs/blocks/caas/caas.css [hunk 2]
```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
...
+button.collapse-panel {
+  border: none;
+  font-size: 18px;
+  width: 26px;
+  height: 32px;
+  cursor: pointer;
+}
+
+button.collapse-panel:hover {
+background:#666;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:543]: Space should be left between property and its value

File: libs/blocks/caas/caas.css [hunk 3]
```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
...
+  grid-template-columns: 20px 20px 1fr;
+}
+
+.panel-collapsed dt.title span {
+  color: #333;
+  padding: 0 3px
+}
+
+.panel-collapsed button {
+cursor:pointer;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:567]: Indentation missing

File: libs/blocks/caas/caas.css [hunk 4]
```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
...
+  z-index: -1;
+}
+
+.panel-collapsed .tool-content {
+  grid-template-columns: 20px 20px 1fr;
+}
+
+.panel-collapsed dt.title span {
+  color: #333;
+  padding: 0 3px
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:563]: Semicolon missing

File: libs/blocks/caas/caas.css [hunk 5]
```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
...
+}
+
+.tool-content {
+  transition: all 400ms;
+  grid-template-columns: 340px 20px 1fr;
+  gap: 6px;
+}
+
+.panel-collapsed .config-panel {
+  z-index: -1;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:554]: `z-index` works only for elements with a position other than `static` or if another stacking context property is used, which I don't see applied in this file, is is added in some other place?

**Review by overmyheadandbody — COMMENTED** [review]
Please add PR labels as well, as described in https://github.com/adobecom/milo/wiki/PR-Labels#required-labels

File: libs/blocks/caas/caas.css (hunk 5 above)
Comment (cmiqueo) [reply libs/blocks/caas/caas.css:554]: The element does not have static position. 
z-index in this context is preventing the mouse events (mouse over and clicks) to fire.

File: libs/blocks/caas/caas.css (hunk 1 above)
Comment (cmiqueo) [reply libs/blocks/caas/caas.css]: done

File: libs/blocks/caas/caas.css (hunk 2 above)
Comment (cmiqueo) [reply libs/blocks/caas/caas.css]: done.

File: libs/blocks/caas/caas.css (hunk 4 above)
Comment (cmiqueo) [reply libs/blocks/caas/caas.css]: done

File: libs/blocks/caas/caas.css (hunk 5 above)
Comment (overmyheadandbody) [reply libs/blocks/caas/caas.css:553]: Then why not use `pointer-events: none` to make the intention clear?

**Review by chrischrischris — APPROVED** [review]

**Pull request merged**